src/main/webapp/WEB-INF/views/car/addDriver.jsp

```diff
@@ -0,0 +1,70 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: glebo
+  Date: 03.02.2023
+  Time: 10:33
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Adding Driver to Car</title>
+</head>
+<body>
+<h1>Please fill in the form</h1>
+
+<form method="post" action="${pageContext.request.contextPath}/cars/drivers/add">
+    Car_id <input type="text" name="car_id"><br>

```

Comment:
![image](https://user-images.githubusercontent.com/118391929/216571863-f2b50191-bfde-46bc-aba8-a41f35e73809.png)
You need to add it everywhere